parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -91,7 +91,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("kintsugi-parachain"),
     impl_name: create_runtime_str!("kintsugi-parachain"),
     authoring_version: 1,
-    spec_version: 4,
+    spec_version: 6,
```

Comment:
Why do we skip 5? I am not sure we can upgrade from 4 to 6, I vaguely remember there being some check that the spec versions only differ by one

Reply:
We didn't, this was included in the hotfix for 1.4.2 which we forgot to do before.